server/spec/hypervisor_check_in_spec.rb

```diff
@@ -288,6 +288,20 @@ def check_hypervisor_consumer(consumer, expected_host_uuid, expected_guest_ids)
     end.should raise_exception(RestClient::BadRequest)
   end
 
+  # simple end to end test
+  it 'should allow virt-who to report hypervisor data' do
+    virtwho = create_virtwho_client(@user)
+    json ="{\"hypervisors\":" +
+          "[{" +
+          "\"name\" : \"hypervisor_999\"," +
+          "\"hypervisorId\" : {\"hypervisorId\":\"uuid_999\"}," +
+          "\"guestIds\" : [{\"guestId\" : \"guestId_1_999\"}]" +
+          "}]}";
```

Comment:
We should test with some facts in there too, the major ones like sockets and such.